### PR TITLE
Cypress/E2E: Add TM4J IDs

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Add TM4J IDs to already automated test cases

### CODE REVIEW

**Review by srkgupta — APPROVED** [review]
LGTM.

File: e2e/cypress/integration/system_console/revoke_all_sessions_spec.js [hunk 1]
```diff
@@ -13,7 +13,7 @@
 import * as TIMEOUTS from '../../fixtures/timeouts';
 import {getAdminAccount} from '../../support/env';
 
-describe('SC17020 - Revoke All Sessions from System Console', () => {
+describe('MM-T940 - Users - Revoke all sessions', () => {
```
Comment (prapti) [inline e2e/cypress/integration/system_console/revoke_all_sessions_spec.js:16]:
```suggestion
describe('MM-T940 Users - Revoke all sessions', () => {
```

File: e2e/cypress/integration/team_settings/remove_team_icon_spec.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ describe('Teams Settings', () => {
         });
     });
 
-    it('TS14632 Remove Team Icon', () => {
+    it('MM-T391 Remove Team Icon', () => {
```
Comment (prapti) [inline e2e/cypress/integration/team_settings/remove_team_icon_spec.js:50]:
```suggestion
    it('MM-T391 Remove team icon', () => {
```

File: e2e/cypress/integration/team_settings/teams_spec.js [hunk 3]
```diff
@@ -35,7 +35,7 @@ describe('Teams Suite', () => {
         });
     });
 
-    it('TS12995 Cancel out of leaving a team', () => {
+    it('MM-T393 Cancel out of leaving a team', () => {
```
Comment (prapti) [inline e2e/cypress/integration/team_settings/teams_spec.js:38]:
```suggestion
    it('MM-T393 Cancel out of leaving team', () => {
```

**Review by prapti — APPROVED** [review]
Approving, but requesting very minor changes to match the text in TM4J test case titles.
Thank you Saturn!

**Comment by saturninoabril** [comment]
Thanks @prapti! Applied all suggestions.

**Pull request merged**